test/Endpoint.spec.js

```diff
@@ -7,20 +7,20 @@ import { Endpoint } from '../src/reduxRest';
 
 describe('Endpoint', () => {
   describe('#_setCSRFHeader()', () => {
-    it('should call a custom setCSRF function', () => {
-      let csrf = sinon.stub();
-      let endpoint = new Endpoint('', {setCSRF: csrf});
-      let request = 'request';
-      endpoint._setCSRFHeader(request);
-      sinon.assert.calledWith(csrf, request);
-    });
+    // it('should call a custom setCSRF function', () => {
```

Comment:
I agree this is a redundant test given the one I added below. You can remove it.